### ISSUE TITLE
Feedback for final deliverable

### ISSUE THREAD

**Issue by ffund** [indianspeedster/edu-ml#1]
### README 

- [x] README/start_here should be the same thing
- [x] shorter title in README
- [x] Add a paragraph saying what the material is and what it's for, explaining generally how the student should work through the material

> In machine learning research papers, sometimes the methodology description seems complete at first glance, but is actually missing some important details that are necessary in order to reproduce the result. This repository contains a fictitious machine learning research paper, and a sequence of accompanying Python notebooks to highlight various choices that can be made to fill in the gaps, and explore how these choices can impact the overall results of the research. 
>
> Our “research paper” is about the impact of data augmentation on few-shot learning for intent classification. We implemented a basic data augmentation strategy with synonym replacement using the HWU64 dataset and a BERT classifier, and the results suggest that synonym replacement as a data augmentation technique leads to only minor improvement in accuracy. 
> 
> In the fictitious paper, we left some of the methodology details ambiguous. When reproducing the results using the accompanying notebooks, the reader follows a “Choose Your Own Adventure” format, selecting a path through a tree, where each node represents ambiguous methodology details and branches out to different choices that are made at that instance. The leaf nodes will represent the final results, providing insights into the magnitude of the differences resulting from each node selection. Some of the choices that the reader makes are -
>
> - what subset of the source dataset to use.
> - some of the details of data pre-processing.
> - some of the details of the synonym replacement data augmentation strategy.
> - some training hyperparameters and the details of the hyperparameter search.

- [x] Add a section called "Run this experiment". It should have the following list items: (1) Read the "paper": "Impact of data augmentation on full few shot learning for intent classification", (2) Reserve resources on Chameleon to reproduce the experiment described in the paper, (3) Follow along with the notebooks provided to run the experiment.

### Launch experiment 

- [x] add a notebook for reserving resource on Chameleon, similar to [this example](https://github.com/teaching-on-testbeds/deep-nets-reproducing/blob/main/reserve.ipynb)
- [x] In the description of the item on Trovi, you will write:

> In machine learning research papers, sometimes the methodology description seems complete at first glance, but is actually missing some important details that are necessary in order to reproduce the result. This repository contains a fictitious machine learning research paper, and a sequence of accompanying Python notebooks to highlight various choices that can be made to fill in the gaps, and explore how these choices can impact the overall results of the research. 
> 
> To run this experiment, launch it on Chameleon, open the "reserve.ipynb" notebook, and follow along from there. 
> 
> This work was supported by the [UCSC OSPO](https://ospo.ucsc.edu/) 2023 Summer of Reproducibility.
- [x] at the top of the reserve.ipynb notebook, mention that the user should have already done "Hello, Chameleon"

### Paper draft

- [x] don't call it "full few shot" every time you refer to it - there is no "partial few shot" in this example so the qualifier is not useful.
- [x] Add full citations.
- [x] Need more related work + cites in intro section.
- [  ] Explain that intent classification is often used on voice assistants, and users may want to add their own custom intents. this is why you often end up in a scenario where you have only a few examples of the "new" intent and a model that is trained on a bunch of other intents. (motivation)
- [x] Explain the question we are trying to answer (in the context of the scenario)
- [x] Methodology should have the following sections: (1) Dataset (2) Data augmentation (3) Scenarios, (4) Training and evaluation.
- [x] Cite dataset, BERT model, etc.... 
- [x] Be careful not to imply that data augmentation strategy is original work.
- [x] Fix description of scenarios
- [x] Label results table
- [x] in results section: discuss the question you set out to answer, no value judgment on the result...
- [x] In discussion: don't speculate, refer to specific examples from data
- [x] Make sure to include all details that are not left out deliberately

### Notebooks

- [x] rename the first from "introduction" to "start_here"
- [x] each notebook should have three "level 2" sections: (1) Your choice for YYY: XXX, (2) Implement your choice, (3) Next step: YYY
- [x] Give reader all the detail they need to make their choice
